File: web-app/packages/api/src/hi/entity/installHi.ts
/**
 * The inserted entity class can be synchronized with the backend
 *
 * property with name
 *
 * @author Elone Hoo <huchengyea@163.com>
 * @since 2022-06-13
 */
export class installHi{
  name:string

  constructor(name:string){
    this.name = name
  }

  public setName(name:string){
    this.name = name
  }

  public getName():string{
    return this.name;
  }

  public toString():string{
    return "InstallHi{" +
    "name='" + this.name + '\'' +
    '}';
  }
}

File: app/basic/src/main/java/com/item/basic/hi/entity/dto/InstallHi.java
package com.item.basic.hi.entity.dto;

import lombok.*;

import java.io.Serializable;

/**
 * <p>an initial insert template</p>
 * <p>Entity classes are constructed using annotations in lombok.</p>
 * @author Elone Hoo <huchengyea@163.com>
 * @since 2022-06-13
 */
@Getter
@Setter
@ToString
@NoArgsConstructor
@AllArgsConstructor
public class InstallHi implements Serializable {
  private String name;

}
